environments/staging/public.yml

```diff
@@ -137,7 +137,7 @@ localsettings:
     - {name: 'ddtrace.contrib.django', host: webworkers}
   LOCAL_MIDDLEWARE:
     - 'django.middleware.security.SecurityMiddleware'
-  SECURE_HSTS_SECONDS: 300
+  SECURE_HSTS_SECONDS: {{ hsts_value_in_seconds }}
```

Comment:
I think you need the quotes here and above for this to be valid yaml. Remember that this is a yaml file where some of the strings are formatted in a way that ansible interprets as variable interpolation, _not_ a jinja2 file that outputs yaml.

Were you having an issue with these values showing up as strings / with quotes around them where they get dumped into templates for the relevant config files?